Source/WebKit/NetworkProcess/cache/NetworkCacheDataGLib.cpp

```diff
@@ -109,10 +110,10 @@ Data concatenate(const Data& a, const Data& b)
     WTF_ALLOW_UNSAFE_BUFFER_USAGE_END
     gsize aLength;
     const auto* aData = g_bytes_get_data(a.bytes(), &aLength);
-    memcpy(data, aData, aLength);
+    memcpySpan(std::span(data, aLength), std::span(static_cast<const uint8_t*>(aData), aLength));
     gsize bLength;
     const auto* bData = g_bytes_get_data(b.bytes(), &bLength);
```

Comment:
I agree with @mcatanzaro but I'd go one step further and say that the reason this is not great is because this PR doesn't go far enough. I do not work on the glib port so please take your lead from developers that maintain this port. However, if this was cocoa code, we'd add a utility wrapper for `g_bytes_get_data` which returns a span, I'd use `MallocSpan<uint8_t>::malloc()` instead of `fastMalloc()` to get a span (though then you have to figure out how to make that work nicely with `g_bytes_new_with_free_func`). And also, use subspan() instead of pointer arithmetics.

Reply:
That sounds like a pretty reasonable (and simple) plan. There are several places using `g_bytes_get_data()` that could benefit from this.